packages/block-editor/src/components/button-block-appender/index.js

```diff
@@ -12,44 +12,40 @@ import { _x, sprintf } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import BlockDropZone from '../block-drop-zone';
 import Inserter from '../inserter';
 
 function ButtonBlockAppender( { rootClientId, className, __experimentalSelectBlockOnInsert: selectBlockOnInsert } ) {
 	return (
-		<>
-			<BlockDropZone rootClientId={ rootClientId } />
```

Comment:
It's not clear why the dropzone was removed from here?

Reply:
It's not removed. It's moved to the more general block list appender.